numpy/__init__.pyi

```diff
@@ -428,11 +437,11 @@ class timedelta64(signedinteger):
     def __sub__(self, other: Union[timedelta64, int]) -> timedelta64: ...
     if sys.version_info[0] < 3:
         @overload
-        def __div__(self, other: timedelta64) -> float: ...
+        def __div__(self, other: timedelta64) -> float64: ...
```

Comment:
A minor bug fix.

Reply:
This code is for python 2 - is there any point keeping it? (Out of scope for this PR, I know)

Reply:
Yeah, might as well remove it (see https://github.com/numpy/numpy/pull/17117/commits/c8c581b383b539667117c71e7e2d320a46440921).